DotBot.py

```diff
@@ -37,6 +37,8 @@ def __init__(self, dotBotId, x, y, floorplan):
         self.next_bump_x          = None  # coordinate the DotBot will bump into next
         self.next_bump_y          = None
         self.next_bump_ts         = None  # time at which DotBot will bump
+        self.newMovementReceived  = True
```

Comment:
don't understand the use of `newMovementReceived`. Here is what I had in mind:
- after it bumps, DotBot sends a notification and schedule to send the same notification 1s later
- if that second expires, it sends a duplicate of the notification and schedule yet another retransmit
- this repeats indefinitely until the DoBot receives a new command from the orchestrator.
We use sequence numbers for both the command and notification to filter duplicates.

Reply:
the newMovementReceived is to know that a new command has been received, i order to stop re-transmitting the same notification. I now understand however that we can know that from the change in sequence number , will modify accordingly.